### PR TITLE
Promote awareness of SwaggerEditor@Next and add About Section

### LINKED ISSUE #3367: Promote usage/awareness of SwaggerEditor@next by offering ability to navigate from existing editor version
Make existing editor users aware a new version exists directly within the current editor version. Additionally, tie off some SEO loose ends by adding an `About` section which can aid navigation to product pages and documentation.

**Rough Mock up**
image (19)

### PR DESCRIPTION
**Pull request by frankkilcommins**
### Description
- Add new topbar 'About' option with links to: Swagger Editor, Product page, Swagger Editor Docs, GitHub repo
- Add button to topbar to promote Trying new editor version
- Change home link of Swagger Editor icon to navigate to the Swagger Editor product page

### Motivation and Context
Promote usage and awareness of new editor and address some SEO low hanging fruit

Fixes #3367

### Screenshot

Screenshot_2022-08-17_12-39-11

### How Has This Been Tested?
- Added Cypress test to validate the new topbar components are loaded
- Manually across Chrome, Edge and Firefox

### My PR contains... 
- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [ ] Improvements (misc. changes to existing features)
- [x] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [ ] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [x] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/standalone/topbar/topbar.jsx [hunk 1]
```diff
@@ -387,6 +387,14 @@ export default class Topbar extends React.Component {
               { this.state.clients
                   .map((cli, i) => <li key={i}><button type="button" onClick={() => this.downloadGeneratedFile("client", cli)}>{cli}</button></li>) }
             </DropdownMenu> : null }
+            <DropdownMenu {...makeMenuOptions("About")}>
+              <li><a href="https://swagger.io/tools/swagger-editor/" target="_blank" rel="noreferrer noopener">About Swagger Editor</a></li>
```
Comment (char0n) [inline src/standalone/topbar/topbar.jsx:391]:
```suggestion
              <li><Link href="https://swagger.io/tools/swagger-editor/" target="_blank">About Swagger Editor</Link></li>
```

File: src/standalone/topbar/topbar.jsx [hunk 2]
```diff
@@ -387,6 +387,14 @@ export default class Topbar extends React.Component {
...
+              <li><a href="https://swagger.io/docs/open-source-tools/swagger-editor/" target="_blank" rel="noreferrer noopener">View Docs</a></li>
```
Comment (char0n) [inline src/standalone/topbar/topbar.jsx:392]:
```suggestion
              <li><Link href="https://swagger.io/docs/open-source-tools/swagger-editor/" target="_blank">View Docs</Link></li>
```

File: src/standalone/topbar/topbar.jsx [hunk 3]
```diff
@@ -387,6 +387,14 @@ export default class Topbar extends React.Component {
...
+              <li><a href="https://github.com/swagger-api/swagger-editor" target="_blank" rel="noreferrer noopener">View on GitHub</a></li>
```
Comment (char0n) [inline src/standalone/topbar/topbar.jsx:393]:
```suggestion
              <li><Link href="https://github.com/swagger-api/swagger-editor" target="_blank">View on GitHub</Link></li>
```

File: src/standalone/topbar/topbar.jsx [hunk 4]
```diff
@@ -387,6 +387,14 @@ export default class Topbar extends React.Component {
...
+            </DropdownMenu>
+            <Link href="https://editor-next.swagger.io/" className="new-editor-cta" target="_blank" rel="noreferrer noopener">
```
Comment (char0n) [inline src/standalone/topbar/topbar.jsx:395]: `rel="noreferrer noopener"` attribute is added automatically by the `<Link>` component.

```suggestion
            <Link href="https://editor-next.swagger.io/" className="new-editor-cta" target="_blank">
```

**Review by char0n — CHANGES_REQUESTED** [review]

**Review by char0n — APPROVED** [review]
LGTM
